src/app/static/src/tests/components/plots/choroplethData/ancIndicators.test.ts

```diff
@@ -0,0 +1,104 @@
+import {
```

Comment:
This file and all of the files in this folder contain test logic that was previously in `getters.test.ts` and `getter_regionIndicators.test.ts`. I broke the latter up into separate files so that I could get my head around it a bit better, and the setup has been simplified due to the functions no longer being getters, but nothing really new here.